doc/swss-schema.md

```diff
@@ -620,7 +622,36 @@ Equivalent RedisDB entry:
     12) "0"
     127.0.0.1:6379>
 
-### Configuration files
+
+## Configuration DB schema
+
+### WARM\_RESTART
+    ;Stores system warm start configuration
+    ;Status: work in progress
+
+    key             = WARM_RESTART:name         ; name is the name of SONiC docker or "system" for global configuration
+    enable          = "true" / "false"          ; default value as false
+    timer           = 1*4DIGIT                  ; timer to determine when reconciliation is done, in seconds.
```

Comment:
add comment that "not all docker will need the reconciliation timer, for example if BGP docker use GR then as soon as the EOR message is received, the reconciliation is finished."

Reply:
A couple of points here...

* The bgp-restartability solution that i've been working on does not rely on EOR functionality -- neither FRR nor Quagga are capable today of generating this message from BGP down to Zebra.

* Even if we had an EOR message coming to fpmSyncd, we would still need a timer to make sure that the reconciliation process is always executed after a reasonable amount of time has elapsed -- the EOR message could get lost or not being generated due to a bug or diverse reasons.

Reply:
Btw, as discussed, i'm fine with this docker/feature-specific timer.

Reply:
neighborsyncd will need the timer as well,  this timer is not necessarily docker specific, but more process specific.

BTW: Should we keep the table key/fields definition in some file, so we know what exactly the tables are supported?

Reply:
If the timer is more process specific,  some update here is necessary since the key here is either "system" or docker name.  For now, we don't want to have support for process level warm restart knob(?) .

I guess this swss-schema.md file is a good place to keep the key/fields definitions for all the DBs?

Reply:
Looks good to me.

I think we need put the table key/fields definitions in the code (e,g, .h file) as well,  so it is synced by all code. This can be done later.